README.md

```diff
@@ -49,7 +49,7 @@ app.post('/graphql', graphqlHTTP({
 
 app.get('/graphql', graphqlHTTP({
   schema: MyGraphQLSchema,
-  graphiql: true // set it to false to disable the GraphiQL interface
```

Comment:
I think the naming and preceding context makes it perfectly clear without the comment.

Without the comment, what would you expect `graphiql: false` to do? What would anybody expect it to do?